### ISSUE TITLE
fork hydehyde theme to avoid the many tweaks

### ISSUE THREAD

**Issue by bric3** [bric3/bric3.github.io#16]
Too many tweak on hydehyde theme

**Comment by bric3** [comment]
Forked there https://github.com/bric3/arkey-hugo-theme

**Comment by bric3** [comment]
**Procedure**

Extract relevant Hugo folders (`archetypes`, `layouts`, `assets`) that needs to be merged in the theme repo.

```shell
$ git clone git@github.com:bric3/bric3.github.io.git theme-tweak-extraction

$ cd theme-tweak-extraction

$ git remote rm origin

$ brew install git-filter-repo

$ git filter-repo --path layouts --path assets --path archetypes --force
Parsed 455 commits
New history written in 0.07 seconds; now repacking/cleaning...
Repacking your repo and cleaning out old unneeded objects
HEAD is now at cf4741c6 Render table of content - Fixes #12
Enumerating objects: 850, done.
Counting objects: 100% (850/850), done.
Delta compression using up to 8 threads
Compressing objects: 100% (357/357), done.
Writing objects: 100% (850/850), done.
Total 850 (delta 430), reused 846 (delta 429), pack-reused 0
Completely finished after 0.35 seconds.
```

Had to clean old jekyll folders 

```
2020-03-30 18:54 +0200 Brice Dutheil o <old-jekyll-website> Add better build experience with make jdk.jshell
2020-03-25 00:30 +0100 Brice Dutheil o Add Java map creation expression
2020-03-13 18:00 +0100 Brice Dutheil o Debug jshell in IntelliJ IDEA
2020-03-12 18:40 +0100 Brice Dutheil o Adds OpenJDK browsing
2018-06-18 12:17 +0200 Brice Dutheil o Minikube article
2017-04-30 12:14 +0200 Brice Dutheil o Adds Kafka walking skeleton article
2016-06-09 18:51 +0200 Brice Dutheil o First import
```

```shell
$ git rebase --interactive 6c3832e92f8e16373b61811340bdda306e85b556
```

Then merge in the theme fork:

```shell
$ git remote add extracted ../../../theme-tweak-extraction

$ git remote update
Fetching origin
Fetching extracted
remote: Enumerating objects: 788, done.
remote: Counting objects: 100% (788/788), done.
remote: Compressing objects: 100% (361/361), done.
remote: Total 788 (delta 370), reused 788 (delta 370), pack-reused 0
Receiving objects: 100% (788/788), 180.01 KiB | 4.62 MiB/s, done.
Resolving deltas: 100% (370/370), done.
From ../../../theme-tweak-extraction
 * [new branch]      hugo-sources -> extracted/hugo-sources

$ git merge --allow-unrelated-histories extracted/hugo-sources --no-ff --stat --strategy-option=theirs
CONFLICT (add/add): Merge conflict in layouts/portfolio/list.html
Auto-merging layouts/portfolio/list.html
CONFLICT (add/add): Merge conflict in layouts/partials/sidebar/social.html
Auto-merging layouts/partials/sidebar/social.html
CONFLICT (add/add): Merge conflict in layouts/partials/sidebar.html
Auto-merging layouts/partials/sidebar.html
CONFLICT (add/add): Merge conflict in layouts/partials/posts-list.html
Auto-merging layouts/partials/posts-list.html
CONFLICT (add/add): Merge conflict in layouts/partials/pagination.html
Auto-merging layouts/partials/pagination.html
CONFLICT (add/add): Merge conflict in layouts/partials/page-single/post-meta.html
Auto-merging layouts/partials/page-single/post-meta.html
CONFLICT (add/add): Merge conflict in layouts/partials/page-single/post-comment.html
Auto-merging layouts/partials/page-single/post-comment.html
CONFLICT (add/add): Merge conflict in layouts/partials/page-single/footer.html
Auto-merging layouts/partials/page-single/footer.html
CONFLICT (add/add): Merge conflict in layouts/partials/page-single/content.html
Auto-merging layouts/partials/page-single/content.html
CONFLICT (add/add): Merge conflict in layouts/partials/page-list/content.html
Auto-merging layouts/partials/page-list/content.html
CONFLICT (add/add): Merge conflict in layouts/partials/highlight-js.html
Auto-merging layouts/partials/highlight-js.html
CONFLICT (add/add): Merge conflict in layouts/partials/header/styles.html
Auto-merging layouts/partials/header/styles.html
CONFLICT (add/add): Merge conflict in layouts/partials/header/styles-highlight.html
Auto-merging layouts/partials/header/styles-highlight.html
CONFLICT (add/add): Merge conflict in layouts/partials/header/meta.html
Auto-merging layouts/partials/header/meta.html
CONFLICT (add/add): Merge conflict in layouts/partials/header/font-awesome-css.html
Auto-merging layouts/partials/header/font-awesome-css.html
CONFLICT (add/add): Merge conflict in layouts/partials/footer/font-awesome-js.html
Auto-merging layouts/partials/footer/font-awesome-js.html
CONFLICT (add/add): Merge conflict in layouts/index.html
Auto-merging layouts/index.html
CONFLICT (add/add): Merge conflict in layouts/about/single.html
Auto-merging layouts/about/single.html
CONFLICT (add/add): Merge conflict in layouts/_default/single.html
Auto-merging layouts/_default/single.html
CONFLICT (add/add): Merge conflict in layouts/_default/list.html
Auto-merging layouts/_default/list.html
CONFLICT (add/add): Merge conflict in layouts/_default/baseof.html
Auto-merging layouts/_default/baseof.html
CONFLICT (add/add): Merge conflict in layouts/404.html
Auto-merging layouts/404.html
CONFLICT (add/add): Merge conflict in assets/scss/hyde-hyde/_variables.scss
Auto-merging assets/scss/hyde-hyde/_variables.scss
CONFLICT (add/add): Merge conflict in assets/scss/hyde-hyde/_taxonomies.scss
Auto-merging assets/scss/hyde-hyde/_taxonomies.scss
CONFLICT (add/add): Merge conflict in assets/scss/hyde-hyde/_sidebar.scss
Auto-merging assets/scss/hyde-hyde/_sidebar.scss
CONFLICT (add/add): Merge conflict in assets/scss/hyde-hyde/_responsive.scss
Auto-merging assets/scss/hyde-hyde/_responsive.scss
CONFLICT (add/add): Merge conflict in assets/scss/hyde-hyde/_misc.scss
Auto-merging assets/scss/hyde-hyde/_misc.scss
CONFLICT (add/add): Merge conflict in assets/scss/hyde-hyde/_code.scss
Auto-merging assets/scss/hyde-hyde/_code.scss
CONFLICT (add/add): Merge conflict in assets/scss/hyde-hyde/_base.scss
Auto-merging assets/scss/hyde-hyde/_base.scss
CONFLICT (add/add): Merge conflict in assets/scss/hyde-hyde.scss
Auto-merging assets/scss/hyde-hyde.scss
CONFLICT (add/add): Merge conflict in assets/scss/hugo-toc.scss
Auto-merging assets/scss/hugo-toc.scss
Merge made by the 'recursive' strategy.
 archetypes/posts.adoc                          |   17 ++++
 archetypes/posts.md                            |   19 ++++
 assets/icons/linkedin-brands.svg               |    1 +
 assets/icons/twitter-brands.svg                |    1 +
 assets/icons/youtube-brands.svg                |    1 +
 assets/scss/hugo-toc.scss                      |  215 ++++++++++++++++++++++++++++++++++++++----
 assets/scss/hyde-hyde-dark.scss                |   11 +++
 assets/scss/hyde-hyde-dark/_asciidark.scss     |  495 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 assets/scss/hyde-hyde-dark/_base.scss          |  214 ++++++++++++++++++++++++++++++++++++++++++
 assets/scss/hyde-hyde-dark/_blockswitch.scss   |   40 ++++++++
 assets/scss/hyde-hyde-dark/_gist.scss          |  510 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 assets/scss/hyde-hyde-dark/_mixins.scss        |    9 ++
 assets/scss/hyde-hyde-dark/_responsive.scss    |   58 ++++++++++++
 assets/scss/hyde-hyde-dark/_taxonomies.scss    |   22 +++++
 assets/scss/hyde-hyde-dark/_variables.scss     |   61 ++++++++++++
 assets/scss/hyde-hyde.scss                     |    2 +
 assets/scss/hyde-hyde/_asciidoctor.scss        | 1027 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 assets/scss/hyde-hyde/_base.scss               |  371 +++++++++++++++++++++++++++++++++++++++++++++++++++++++-----------------
 assets/scss/hyde-hyde/_blockswitch.scss        |   61 ++++++++++++
 assets/scss/hyde-hyde/_code.scss               |   70 +++++++-------
 assets/scss/hyde-hyde/_misc.scss               |   42 ++++++++-
 assets/scss/hyde-hyde/_responsive.scss         |  241 ++++++++++++++++++++++++++++++++++++++++++++++-
 assets/scss/hyde-hyde/_sidebar.scss            |   83 +++++++++++++++--
 assets/scss/hyde-hyde/_taxonomies.scss         |    6 +-
 assets/scss/hyde-hyde/_variables.scss          |   74 +++++++++++++--
 layouts/404.html                               |  494 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++-
 layouts/_default/baseof.html                   |    2 +-
 layouts/_default/list.html                     |    4 +-
 layouts/_default/rss.xml                       |   41 ++++++++
 layouts/_default/section.html                  |   11 +++
 layouts/_default/single-baseof.html            |   13 +++
 layouts/_default/single.html                   |    8 +-
 layouts/_default/sitemap.xml                   |   24 +++++
 layouts/about/single.html                      |    6 +-
 layouts/index.html                             |    8 +-
 layouts/partials/footer/font-awesome-js.html   |    9 +-
 layouts/partials/footer/mathjax-js.html        |   25 +++++
 layouts/partials/github-edit-link.html         |   10 ++
 layouts/partials/head.html                     |   35 +++++++
 layouts/partials/header/font-awesome-css.html  |    4 +-
 layouts/partials/header/meta.html              |   50 +++++++++-
 layouts/partials/header/styles-highlight.html  |   18 ++--
 layouts/partials/header/styles.html            |    3 +
 layouts/partials/highlight-js.html             |   12 +--
 layouts/partials/page-list/content.html        |   78 +++++++++++++++-
 layouts/partials/page-single/content.html      |   10 +-
 layouts/partials/page-single/footer.html       |   89 ++++++++++++++++++
 layouts/partials/page-single/header.html       |    3 +
 layouts/partials/page-single/post-comment.html |   39 ++++++--
 layouts/partials/page-single/post-meta.html    |   18 +++-
 layouts/partials/pagination.html               |   10 +-
 layouts/partials/posts-list.html               |   18 ++--
 layouts/partials/posts/content.html            |   16 ++++
 layouts/partials/sidebar.html                  |   13 +--
 layouts/partials/sidebar/social.html           |   22 +----
 layouts/partials/tags/content.html             |   76 +++++++++++++++
 layouts/portfolio/list.html                    |    4 +-
 layouts/posts/list.html                        |   11 +++
 layouts/shortcodes/amzn.html                   |   49 ++++++++++
 layouts/shortcodes/c.html                      |    5 +
 layouts/shortcodes/draftNotes.html             |   11 +++
 layouts/shortcodes/fab.html                    |    1 +
 layouts/shortcodes/fas.html                    |    1 +
 layouts/shortcodes/mark.html                   |    6 ++
 layouts/shortcodes/wrapTable.html              |   19 ++++
 layouts/tags/list.html                         |   11 +++
 66 files changed, 4682 insertions(+), 256 deletions(-)
 create mode 100644 archetypes/posts.adoc
 create mode 100644 archetypes/posts.md
 create mode 100644 assets/icons/linkedin-brands.svg
 create mode 100644 assets/icons/twitter-brands.svg
 create mode 100644 assets/icons/youtube-brands.svg
 create mode 100644 assets/scss/hyde-hyde-dark.scss
 create mode 100644 assets/scss/hyde-hyde-dark/_asciidark.scss
 create mode 100644 assets/scss/hyde-hyde-dark/_base.scss
 create mode 100644 assets/scss/hyde-hyde-dark/_blockswitch.scss
 create mode 100644 assets/scss/hyde-hyde-dark/_gist.scss
 create mode 100644 assets/scss/hyde-hyde-dark/_mixins.scss
 create mode 100644 assets/scss/hyde-hyde-dark/_responsive.scss
 create mode 100644 assets/scss/hyde-hyde-dark/_taxonomies.scss
 create mode 100644 assets/scss/hyde-hyde-dark/_variables.scss
 create mode 100644 assets/scss/hyde-hyde/_asciidoctor.scss
 create mode 100644 assets/scss/hyde-hyde/_blockswitch.scss
 create mode 100644 layouts/_default/rss.xml
 create mode 100644 layouts/_default/section.html
 create mode 100644 layouts/_default/single-baseof.html
 create mode 100644 layouts/_default/sitemap.xml
 create mode 100644 layouts/partials/footer/mathjax-js.html
 create mode 100644 layouts/partials/github-edit-link.html
 create mode 100644 layouts/partials/head.html
 create mode 100644 layouts/partials/page-single/header.html
 create mode 100644 layouts/partials/posts/content.html
 create mode 100644 layouts/partials/tags/content.html
 create mode 100644 layouts/posts/list.html
 create mode 100644 layouts/shortcodes/amzn.html
 create mode 100644 layouts/shortcodes/c.html
 create mode 100644 layouts/shortcodes/draftNotes.html
 create mode 100644 layouts/shortcodes/fab.html
 create mode 100644 layouts/shortcodes/fas.html
 create mode 100644 layouts/shortcodes/mark.html
 create mode 100644 layouts/shortcodes/wrapTable.html
 create mode 100644 layouts/tags/list.html
```
